Accessibility.md

```diff
@@ -0,0 +1,32 @@
+# Accessibility
+
+## Motivation
+
+Why we should we focus on a11y?
+What are the benefits?
+Maybe some cons?
+
+## Screen reader
+
+Mac's Voice Over (and alternatives)
+Window's screen readers
+
+## Semantics
+
+What is semantics? (short paragrphs)
+List of semantic elements (or link to extrnal list)
+How to structure a page?
+
```

Comment:
What about Accessible Colors?

Storybook also has a plugin for a11y https://storybook.js.org/addons/@storybook/addon-a11y maybe its cool thing to mention.

Reply:
> What about Accessible Colors?

I am not really sure if this is something we should really worry about. I mean, yeah, if we notice that the contrast is low (white bgColor and light gray fontColor) we should mention this, but often we just implement what we get from designers.

Maybe we can add the A11y section to the pre-production checklist?

Reply:
> Storybook also has a plugin for a11y https://storybook.js.org/addons/@storybook/addon-a11y maybe its cool thing to mention.

I agree, this looks cool! Maybe we add a new section with extensions, addons, plugins for storybook, browsers, IDEs? What do you think?